better_longevity_ideas.py

```
@@ -0,0 +1,64 @@
+            phase_result = result.phases.get(phase_name)
+            if phase_result and phase_result.generated_ideas:
+                for idea in phase_result.generated_ideas:
+                    all_ideas.append({
+                        'focus': problem_set['focus'],
+                        'phase': phase_name,
+                        'content': idea.content
+                    })
+                    print(f"• {idea.content}")
+    
+    print(f"\n🎉 Generated {len(all_ideas)} specific longevity ideas!")
+    print("Focus areas covered:", set(idea['focus'] for idea in all_ideas))
```

Ruff (0.11.9)

Unnecessary generator (rewrite as a set comprehension)

Rewrite as a set comprehension

(C401)